packages/ui/src/working-groups/components/WorkingGroupImagePlaceholder.tsx

```diff
@@ -0,0 +1,335 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Colors, Transitions } from '../../common/constants'
+
+export interface WGImagePlaceholderProps {
+  className?: string
+  groupName: string | undefined
+}
+
+export const WorkingGroupImagePlaceholder = ({ className, groupName }: WGImagePlaceholderProps) => {
```

Comment:
```suggestion
export const WorkingGroupImage = ({ className, groupName }: WGImagePlaceholderProps) => {
```

You can drop the "placeholder" as we will always use these images.